internal/webhook/webhook_test.go

```diff
@@ -272,15 +267,15 @@ func TestSpiffeEnableWebhook_Handle(t *testing.T) {
 			validatePod: nil,
 		},
 		{
-			name:           "spiffe.cofide.io/enabled: true, CSI volume already exists",
-			podAnnotations: map[string]string{constants.EnabledAnnotation: "true"},
+			name:           "No pod annotation, CSI volume already exists",
```

Comment:
The old test covered the case where CSI injection was requested but it already exists. Should we add another case to cover that?

Reply:
Good spot. Here it is: https://github.com/cofide/spiffe-enable/pull/20/commits/b6851f55677d065670938ce59bd8bfa81fb48716.